index.bs

```diff
@@ -339,13 +340,60 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 :: A user agent implementing, in conjunction with the underlying platform, the [=Web Authentication API=] and algorithms
     given in this specification, and handling communication between [=authenticators=] and [=[RPS]=].
 
+: <dfn>Credential ID</dfn>
+:: A probabilistically-unique [=byte sequence=] identifying a [=public key credential source=].
+
+    Credential IDs are generated by [=authenticators=] in two forms:
+    1. Bytes with at least 100 bits of entropy, or
+    1. The [=public key credential source=], without its [=public key credential source/id=] item, encrypted so only the owning
+        authenticator can decrypt it.
+
+    [=[RPS]=] don't need to distinguish these two kinds of Credential IDs.
+
 : <dfn>Credential Public Key</dfn>
 :: The public key portion of an [=[RP]=]-specific <dfn>credential key pair</dfn>, generated by an [=authenticator=] and
     returned to an [=[RP]=] at [=registration=] time (see also [=public key credential=]). The private key portion of the
     [=credential key pair=] is known as the <dfn>credential private key</dfn>. Note that in the case of [=self
     attestation=], the [=credential key pair=] is also used as the [=attestation key pair=], see [=self attestation=]
     for details.
 
+: <dfn>Public Key Credential Source</dfn>
```

Comment:
Public Key Credential Source is also already defined in the spec.  I'm going to stop reviewing this PR at this point because master clearly needs to be merged into this PR and the conflicts resolved before it can be meaningfully reviewed.